src/views/pacbio/PacbioLibraryBatchCreate.vue

```diff
@@ -0,0 +1,312 @@
+              />
+            </div>
+
+            <div class="w-full h-full border-2">
+              <div class="w-full h-full p-4 space-y-4 rounded-md">
+                <traction-heading level="4" show-border> Preview Barcodes </traction-heading>
+                <div class="h-full">
+                  <div class="space-x-4 pb-4 flex flex-row">
+                    <traction-button
+                      id="print-button"
+                      class="grow"
+                      :disabled="isPrintDisabled"
+                      theme="printRed"
+                      @click="onPrintAction"
+                    >
+                      Print Labels
+                    </traction-button>
+                  </div>
+
+                  <div tag="article" class="mb-2 text-black text-left">
+                    <div class="flex flex-col rounded p-4">
+                      <ul
+                        v-if="printBarcodes.length"
+                        id="list-barcodes-to-print"
+                        class="space-y-2 justify-center text-center"
+                      >
+                        <li
+                          v-for="barcode in printBarcodes"
+                          :key="barcode"
+                          class="text-sm text-sp font-bold"
+                        >
+                          {{ barcode }}
+                        </li>
+                      </ul>
+                      <div
+                        v-else
+                        data-type="empty-barcodes"
+                        class="justify-center text-center text-black text-sm font-bold"
+                      >
+                        No Barcodes to Print Yet
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </DataFetcher>
+</template>
+
+<script setup>
+/**
+ * PacbioLibraryBatchCreate.vue
+ *
+ * This component is used to create a library batch from a csv file and tag set
+ * and print labels for the created libraries
+ *
+ * @module PacbioLibraryBatchCreate
+ */
+import { usePacbioRootStore } from '@/stores/pacbioRoot.js'
+import { usePrintingStore } from '@/stores/printing.js'
+import { usePacbioLibraryBatchCreateStore } from '@/stores/pacbioLibraryBatchCreate.js'
+import { computed, ref, reactive } from 'vue'
+import DataFetcher from '@/components/DataFetcher.vue'
+import { eachRecord } from '@/lib/csv/pacbio.js'
+import useAlert from '@/composables/useAlert.js'
+import usePacbioLibraryPrint from '@/composables/usePacbioLibraryPrint.js'
+
+const pacbioRootStore = usePacbioRootStore()
+const pacbioLibraryBatchCreateStore = usePacbioLibraryBatchCreateStore()
+const { printers, fetchPrinters } = usePrintingStore()
+const { showAlert } = useAlert()
+const { printLabels } = usePacbioLibraryPrint()
+
+const defaultTagSetName = 'Pacbio_96_barcode_plate_v3'
```

Comment:
Something for later but it is likely that the default tag set will change so it might be worth adding this as a flag in the tag set table. Maybe we need a story to add this to the service.